.github/workflows/build-ami.yml

```diff
@@ -0,0 +1,62 @@
+name: Build AWS Neuron AMI
+
+on:
+    push:
+      branches:
+        - main
+      paths:
+        - 'aws-ami-creation/**'
```

Comment:
We should add "pull_request" too otherwise you cannot validate if the changes work before merging.